app/controllers/users_controller.rb

```diff
@@ -1,8 +1,19 @@
 class UsersController < ApplicationController
-  
+  # Manual Authentication with Devise feature
+  # def index
+  #   if current_user.user_type == 'admin'
+  #   @users = User.order(created_at: :asc)
+  #   else
+  #     redirect_to root_path, alert: 'not_authorized'
+  #   end 
+  # end
+
   def index
-    @users = User.order(created_at: :asc) 
+    
+    @users = User.order(created_at: :asc)
```

Comment:
We are not using paggy here?

Reply:
> We are not using paggy here?

No I have just used on article index page